### DISCUSSION TITLE
Do you have any power supply suggestions?

### DISCUSSION THREAD

**Discussion by erdempome** [Chia-Network/chia-blockchain#8575] [Support]
Do you have any power supply suggestions?  for more hdds

**Comment by loppefaaret** [comment]
perhps try and look through the https://www.reddit.com/r/chia/ or join our keybase chat https://keybase.io/team/chia_network.public

**Comment by BrandtH22** [comment]
@erdempome , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket
